bootloader/bin/bootstrap.sh

```diff
@@ -273,10 +273,16 @@ get_donor_selection() {
 
 boot_target() {
   local target="$1"
+  local name="$2"
 
   echo "moving mounts to newroot"
   mkdir /newroot
-  mount $target /newroot
+  if [ $name = 'luks2' ]; then
```

Comment:
Don't hardcode the name here. You should instead detect if a partition is LUKS at runtime with cryptsetup.